src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -269,6 +270,7 @@ object ClusterInitValues {
       proxyConfig.proxyServerName,
       clusterRequest.jupyterExtensionUri.map(_.toUri).getOrElse(""),
       clusterRequest.jupyterUserScriptUri.map(_.toUri).getOrElse(""),
+      GcsPath(stagingBucket, GcsObjectName("userscript_output.txt")).toUri,
```

Comment:
What if we named the output file based on the user script name? So if my script was named `foo.sh` the output could be `foo_output.log`?

Reply:
Then again maybe a consistent name is good so the UI is able to easily find it...

Reply:
Will we be allowing users to specify multiple user scripts (for example, when we introduce more tools?). I thought keeping it consistent would make it easier to follow, but if we will eventually support multiple userscripts it might be a good idea to make the name specific

Reply:
Good point. For now we just support 1 user script so maybe we cross that bridge when we come to it.